cypress/support/commands.js

```diff
@@ -1,22 +1,77 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import { SELECTORS } from '../constants/IdConstants';
-import {
-  PAGE_NAME,
-  URL_ROOT
-} from '../constants/TestConstants.js';
-
-const urlRegexWithAnySuffix = new RegExp(`^${URL_ROOT}/.*${PAGE_NAME.SCENARIOS}/.*`);
+import { SELECTORS } from '../commons/IdConstants.js';
+import { URL_REGEX } from '../commons/TestConstants.js';
 
 // For this moment, login is handle by clicking on Microsoft Login button
 Cypress.Commands.add('login', () => {
-  cy.intercept('GET', urlRegexWithAnySuffix).as('requestUpdateCurrentScenario1');
+  // Stub PowerBi request
+  cy.intercept('GET', 'http://localhost:3000/api/get-embed-info', {
+    statusCode: 200
+  });
+
+  cy.intercept('GET', URL_REGEX.WITH_UNKNOWN_ID_SCENARIO_SUFFIX).as('requestUpdateCurrentScenario');
 
   cy.get(SELECTORS.login.microsoftLoginButton).click();
 
   // TODO : Remove timeout due to loading page
-  cy.wait('@requestUpdateCurrentScenario1', { timeout: 10_000 });
+  cy.wait('@requestUpdateCurrentScenario', { timeout: 10_000 });
```

Comment:
I've done some local test. The timeout is not needed anymore. You can remove this line (for the moment)